eng/helix/content/RunTests/TestRunner.cs

```diff
@@ -166,9 +166,9 @@ public bool InstallAspNetRefIfNeeded()
             {
                 if (File.Exists(Options.AspNetRef))
                 {
-                    var refPath = $"Microsoft.AspNetCore.App.Ref";
+                    var refPath = $"{Options.DotnetRoot}/packs/Microsoft.AspNetCore.App.Ref/{Options.RuntimeVersion}";
```

Comment:
Previous location worked fine for `TargetingPackTests` but nothing actually compiled against the new ref/ assemblies. This causes problems with stable versions i.e. when assemblies build against 5.0.0 but can't find it (preview builds roll forward).